content/coronavirus_landing_page.yml

```diff
@@ -14,6 +14,8 @@ content:
     link:
       href: /government/publications/coronavirus-outbreak-faqs-what-you-can-and-cant-do/coronavirus-outbreak-faqs-what-you-can-and-cant-do
       text: "Read more about what you can and cannot&nbsp;do"
+      link_text: "Read more about what you can and cannot"
+      link_nowrap_text: "do"
```

Comment:
Shouldn't the last two words be in the non-wrapping span? i.e.
```
link_text: "Read more about what you can and"
link_nowrap_text: "cannot do"
```

Reply:
@andysellick yes... 🤦‍♀️